src/darwin/Framework/CHIP/MTRDeviceOverXPC.h

```diff
@@ -25,11 +25,6 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (instancetype)init NS_UNAVAILABLE;
 + (instancetype)new NS_UNAVAILABLE;
-- (void)subscribeWithQueue:(dispatch_queue_t)queue
-                minInterval:(NSNumber *)minInterval
-                maxInterval:(NSNumber *)maxInterval
-              reportHandler:(void (^)(NSArray * _Nullable value, NSError * _Nullable error))reportHandler
-    subscriptionEstablished:(MTRSubscriptionEstablishedHandler _Nullable)subscriptionEstablishedHandler NS_UNAVAILABLE;
```

Comment:
So this one is a leftover from https://github.com/project-chip/connectedhomeip/commit/153a564928634654576038839f904983ac600fba and the signature is unused now ?

Reply:
Yes, since then subscription was implemented for remoted devices.